docs/vagrant.md

```diff
@@ -11,15 +11,15 @@ Running with Vagrant
 * You can speed up repeated provisioning attempts by mirroring the Splunk package downloads locally:
   1. Download the needed splunk packages locally, in a directory structure mirroring that of download.splunk.com
     * You can find the suffixes of files at the [Splunk download page](http://splunk.com/download)
-    * Copy a download link, it will look like `http://www.splunk.com/page/download_track?file=6.2.5/splunk/linux/splunk-6.2.5-272645-linux-2.6-x86_64.rpm&platform=...`
-    * The `file` query parameter will be the suffix you add onto `http://download.splunk.com/releases/`
-      * In this example the full url is: `http://download.splunk.com/releases/6.2.5/splunk/linux/splunk-6.2.5-272645-linux-2.6-x86_64.rpm`
-      * You will download the file locally to `<Mirror root>/releases/6.2.5/splunk/linux/splunk-6.2.5-272645-linux-2.6-x86_64.rpm
+    * Copy a download link, it will look like `http://www.splunk.com/page/download_track?file=6.3.3/splunk/linux/splunk-6.3.3-f44afce176d0-linux-2.6-x86_64.rpm&platform=...`
+    * The `file` query parameter will be the suffix you add onto `http://download.splunk.com/products/splunk/releases/`
+      * In this example the full url is: `https://download.splunk.com/products/splunk/releases/6.3.3/linux/splunk-6.3.3-f44afce176d0-linux-2.6-x86_64.rpm`
+      * You will download the file locally to `<Mirror root>/releases/6.3.3/splunk/linux/splunk-6.3.3-f44afce176d0-linux-2.6-x86_64.rpm
```

Comment:
Things I should have updated with the switch of URL structures... Mirror will use the same structure as download.splunk.com. `(mirror root)/products/splunk/releases/<version>/...` (assuming we change the mirror root url).

Copy the download link only works for the previous releases anymore... for the most current version you have to inspect the link and you get the full url from the first argument to handleDownload. I wonder if we want to drop these specific instructions to just "Mirror appropriate packages and structure from download.splunk.com